modular_skyrat/modules/expeditionary-corps/code/misc.dm

```diff
@@ -0,0 +1,5 @@
+/obj/machinery/spaceship_navigation_beacon/blackbox
+	name = "Blackbox navigation beacon"
```

Comment:
```suggestion
	name = "blackbox navigation beacon"
```

Reply:
In most cases, the names will be changed to custom names when creating wreck cards. But I'll change it to comply with the standard.